### ISSUE TITLE
Integrating screen readers and keyboard navigation

### ISSUE THREAD

**Issue by alschmiedt** [RaspberryPiFoundation/blockly#3370]
**Is your feature request related to a problem? Please describe.**
Should look at how keyboard navigation interacts with screen readers. 
- Focus should go to  toolbox when navigation opens the toolbox.
- How fields deal with focus 
- Should loose focus when we click on something out of the workspace
I am sure there are more I am just not aware of yet.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by BenHenning** [comment]
Note from https://github.com/google/blockly-keyboard-experimentation/issues/523#issuecomment-2901951222 that this work should include performing an NVDA test, particularly a page-level point-in-time analysis since our use of tabindexes is pretty complicated and could have unintended side effects.

This analysis should probably wait until more of the ARIA context is correct since that will presumably have a big impact on the context and how it's represented.

**Comment by BenHenning** [comment]
Note: I'm treating this as the general "add screen reader support into Blockly" for the sake of work tracking.

**Comment by rachel-fenichel** [comment]
@BenHenning Is this tracking issue still useful, or should we close it in favor of the smaller issues that have already been broken out?

**Comment by maribethb** [comment]
I don't think this tracking bug is still super useful, the entire project board is sort of tracking this issue.

**Comment by BenHenning** [comment]
Discussed with @maribethb over chat. I plan to use this as a general view for the initial screen reader integration into Blockly (i.e. v13), and also a view of all of the work that went into the experimental changes (as a visual parallel to the screen reader document). I'll be tweaking the issue organization over the next few weeks since it's not urgent.

This contrasts a bit from the project board because that also includes bugs and sometimes non-screen reader issues that don't quite fit into the actual "introduce screen reader" project.

**Comment by gonfunko** [comment]
Closing now that v13 is out the door.
